samples/proxy-coredump/daemonset.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: enable-istio-coredumps
+  namespace: istio-system
+  labels:
+    app: enable-istio-coredumps
+spec:
+  selector:
+    matchLabels:
+      app: enable-istio-coredumps
+  template:
+    metadata:
+      labels:
+        app: enable-istio-coredumps
+    spec:
+      hostNetwork: true
+      # hostPID: true
+      # hostIPC: true
+      initContainers:
+        - name: setsysctls
+          command:
+            - sh
+            - -c
+            - sysctl -w kernel.core_pattern=/var/lib/istio/data/core.proxy && ulimit -c unlimited
```

Comment:
So this will save the coredumps where on the node?

Reply:
The problem is the 'where to dump core files' is a node-level setting. But it doesn't somehow put them on the node filesystem, it still puts them in the pod filesystem. So need to move it into a writeable folder.

Note the `readOnlyRootFilesystem` part was obsolete since `/var/lib/istio/data` is a volume now